src/net_processing.cpp

```diff
@@ -3286,6 +3274,24 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
         return true;
     }
 
+    // VeriBlock: if POP is not enabled, ignore POP-related P2P calls
+    if (VeriBlock::isPopActive()) {
+        if(ChainstateActive().IsInitialBlockDownload()) {
+            // ignore all POP-related messages until sync is completed
+            return true;
+        }
+        if(ChainActive().Height() < pfrom->nStartingHeight - 5) {
```

Comment:
Do we have a place to declare constants or parameters like the value 5?

Reply:
it is temporary fix, I do not think we need to do this